src/TwigComponent/doc/index.rst

```diff
@@ -466,29 +476,46 @@ called ``mount()``::
         public string $message;
         public string $type = 'success';
 
-        public function mount(bool $isSuccess = true): void
+        public function mount(): void
         {
-            $this->type = $isSuccess ? 'success' : 'danger';
+            {# ❌ this won't work: at this point, the $type property still has its
+                   default value. Passed properties are not yet available. #}
+            if ('error' === $this->type) {
+                // ...
+            }
         }
 
         // ...
     }
 
-The ``mount()`` method is called just one time: immediately after your
-component is instantiated. Because the method has an ``$isSuccess``
-argument, if we pass an ``isSuccess`` prop when rendering, it will be
-passed to ``mount()``.
+Instead, define the values you need in the ``mount()`` method as arguments::
+
+    #[AsTwigComponent]
+    class Alert
+    {
+        public string $message;
+        public string $type = 'success';
+
+        public function mount(bool $isError = false): void
+        {
+            {# ✅ this works as expected #}
+            if ($isError) {
+                // ...
```

Comment:
Like it was done before, we should keep an example (or document) that we can modify a property here
```suggestion
                $this->type = 'danger';
```